R/aes.r

```diff
@@ -103,7 +103,7 @@ new_aesthetic <- function(x, env = globalenv()) {
 }
 new_aes <- function(x, env = globalenv()) {
   if (!is.list(x)) {
-    abort("`x` must be a list")
+    cli::cli_abort("{.arg x} must be a {.cls list}")
```

Comment:
I don't think we use `.cls` for simple types.

Reply:
hmm... but if we have something like `{.cls {class(x)}}` and `x` is a list, it will get the formatting and thus be inconsistent if we didn't apply it fully for the hardcoded situations...

Reply:
I think that's ok because the type of message usually only comes up in some fallback, where most of the time `x` will be an S3 object.

Reply:
ok - I'll update